docs/guides/getting-started-java-spring.md

```diff
@@ -0,0 +1,255 @@
+
+```
+<repositories>
+   <repository>
+      <releases>
+         <enabled>true</enabled>
+      </releases>
+      <snapshots>
+         <enabled>false</enabled>
+      </snapshots>
+      <id>identity</id>
+      <name>Camunda Identity</name>
+      <url>https://artifacts.camunda.com/artifactory/camunda-identity/</url>
+   </repository>
+</repositories>
+```
+
+2. Add the following dependency to your `pom.xml` file:
+
+```
+<dependency>
+   <groupId>io.camunda</groupId>
+   <artifactId>spring-boot-starter-camunda-sdk</artifactId>
+   <version>8.5.0</version>
+</dependency>
+```
+
+### Configure the Zeebe client
+
+Open your `src/main/resources/application.properties` file, and paste the following snippet to connect to the Self-Managed Zeebe broker:
+
+```
+zeebe.client.broker.grpcAddress=http://127.0.0.1:26500
+zeebe.client.broker.restAddress=http://127.0.0.1:8080
+zeebe.client.security.plaintext=true
+```
+
+### Create a worker
+
+1. In `src/main/java/io/camunda/demo/process_payments/`, create a file called `ChargeCreditCardWorker.java`.
+2. In the file created above, paste the following dependencies and package `package io.camunda.demo.process_payments`:
+
+```java
+package io.camunda.demo.process_payments;
+
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import io.camunda.zeebe.spring.client.annotation.JobWorker;
+import io.camunda.zeebe.spring.client.annotation.Variable;
+```
+
+3. Next, we can decorate our class with `@Component` and instantiate a logger:
+
+```java
+@Component
+public class ChargeCreditCardWorker {
+
+  private final static Logger LOG = LoggerFactory.getLogger(ChargeCreditCardWorker.class);
+```
+
+4. Add the following method and decorate the `chargeCreditCard` method with `@JobWorker`:
+
+```java
+  @JobWorker(type = "charge-credit-card")
+```
+
+5. Specify a `@Variable(name = "totalWithTax") Double totalWithTax` argument into the `chargeCreditCard` method:
+
+```java
+  public Map<String, Double> chargeCreditCard(@Variable(name = "totalWithTax") Double totalWithTax) {
+```
+
+6. Lastly, implement the `chargeCreditCard` method:
+
+```java
+   LOG.info("charging credit card: {}", totalWithTax);
+    return Map.of("amountCharged", totalWithTax);
+```
+
+:::note
+If you want to check your work, visit our [sample repository](https://github.com/camunda/camunda-8-get-started-spring/blob/main/src/main/java/io/camunda/demo/process_payments/ChargeCreditCardWorker.java) with the completed code.
+:::
+
+In your terminal, run `mvn spring-boot:run`, where you should see the `charging credit card` output. In Operate, refresh if needed, and note the payment has executed.
+
+## Step 7: Start a process instance
+
+To kick off your process instance, take the following steps:
+
+1. In `ProcessPaymentsApplication.java`, instantiate a static logger:
+
+```java
+@SpringBootApplication
+
```

Comment:
```suggestion
public class ProcessPaymentsApplication implements CommandLineRunner {
```

There's a line of code missing here. The `@SpringBootApplication` annotation decorates the `ProcessPaymentsApplication` class. The `Logger` declaration happens within the `ProcessPaymentsApplication` class.

We explain below that we're converting the class to a CommandLineRunner, and that might be why you took that line out here, but it will result in non-compiling code. Maybe it means we should describe that we're converting to a CommandLineRunner before doing the other things; maybe also it would be less risk for mistake if we used one code snippet for the entire class.